src/components/list/list.mdx

```diff
@@ -8,23 +8,31 @@ menu: Components
 
 <limel-props name="limel-list" />
 
+## Custom styles
+
+| Class name | Description                                                                                     |
+| -------- | -------------------------------------------------------------------- |
+| `has-striped-rows` | Adds distinct styling and more interactivity to list items |
```

Comment:
The description is very generic, maybe make it a bit more specific like "Adds every other row a darker background" or something like that. What do you think?

Reply:
I like your suggestion. 👌will you add it or should I do some fixups?

Reply:
You can do it :slightly_smiling_face: